primuse/SendIt-React
# #164532524 Protect routes with Private Route

### Description of Task to be completed?
An unauthorized user shouldn't be able to access authenticated routes.
### How should this be manually tested?
- After cloning the repo, ```cd``` into it and run
```npm run start-dev```

- visit ```localhost:8080/dashboard```  to view the dashboard page. You should be redirected to the login page.

### What are the relevant pivotal tracker stories?
[#164532524](https://www.pivotaltracker.com/story/show/164532524)
